src/app/globals.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class Globals {
+  scrolledState = false;
+  role = 'test';
```

Comment:
`role` is this just for test?

Reply:
I'm still experimenting with communication between services. This approach seemed to work fine but ng-test fails :(
I need a proper angular way for this. I'm trying out examples on a sample project. Let me know if you have any links that could be useful :)